README.md

```diff
@@ -0,0 +1,26 @@
+# AngularDockerSampleApp
+
+## Development
+
+1. Clone the repository
+2. Go to the root directory of [angular-docker-sample-app](./angular-docker-sample-app)
+3. Execute the following commands
+```shell
+# install dependencies
+npm install
+
+# start app
+ng serve
+```
+4. Access the app at http://localhost:4200
+
+## Docker
```

Comment:
It might be a little confusing to split steps for setup over two README files, can we consolidate that into one only?

Reply:
The outer readme file has the complete setup
What I can do is delete the inner readme file